docs/reference/ingress/kubernetes-with-cert-manager.md

```diff
@@ -0,0 +1,168 @@
+# Deploying with SSL on Kubernetes using Cert-Manager and Nginx
+
+The quickest way to get automated SSL/TLS certificates for your project is to use [cert-manager][cert-manager]. In this tutorial, we will deploy OpenFaaS using the [Helm Chart][openfaas-helm], [cert-manager][cert-manager], and [nginx-ingress][nginx-ingress]
+
+**RBAC** We will assume that your cluster is configured for [RBAC][k8s-rbac].
+
+## Configure Helm and Tiller
+
+First install Helm and the Tiller [following the instructions provided by Helm][helm-install]
+
+## Create namespaces
+
+```sh
+$ kubectl create namespace openfaas
+$ kubectl create namespace openfaas-fn
+```
+
+## Install nginx-ingress
```

Comment:
Might be good to mention that other Ingress controllers are also valid.

@stefanprodan has a preference for Heptio Contour, so we could link to that.

Reply:
Sounds good, I will add additional links once @stefanprodan let's me know which one would be best